packages/react/src/components/Badge.tsx

```
@@ -1,35 +1,10 @@
+import type { ComponentRef, ComponentType } from "react";
 
-export type BadgeProps = ExtendedComponentProps<
-  typeof ark.div,
-  {
-    size?: BadgeSize;
-    variant?: BadgeVariant;
-  }
->;
-export type BadgeRef = ComponentRef<typeof ark.div>;
+import { ark } from "@ark-ui/react";
+import { badge } from "@sandwich-ui/styled-system/recipes";
```

ESLint

[error] Unable to resolve path to module '`@sandwich-ui/styled-system/recipes`'.

(import-x/no-unresolved)